Jenkinsfile

```diff
@@ -3,6 +3,7 @@ pipeline {
     stages {
         stage('Compile') {
             steps {
+                sh 'sbt reload'
```

Comment:
Since the time I've been looking for this one... Cheers :)

Reply:
Nope it don't work, it only reload existing project declarations without refreshing the clone...

Reply:
Ah damn... tricky ^^'